imageio_ffmpeg/_io.py

```diff
@@ -312,6 +314,8 @@ def write_frames(
             ffmpeg needs depends on CPU speed, compression, and frame size.
         input_params (list): Additional ffmpeg input command line parameters.
         output_params (list): Additional ffmpeg output command line parameters.
+        audio_path (str): A input file path for encoding with an audio stream.
+        audio_codec (str): The audio codec to use if audio_path is provided.
```

Comment:
Does ffmpeg select a default audio codec if this is not provided? Would it be good for us to apply a default?

Reply:
I think that ffmpeg applies defaults per format (https://stackoverflow.com/questions/17793150/default-ffmpeg-codec-when-nothing-is-specified). I also know that webm, for example, supports only a few specific audio codecs that aren't very "popular".

Reply:
Ok, then let's mention that ffmpeg will select a codec if none is provided (or something along that line).